crates/context_aware_config/src/api/default_config/handlers.rs

```diff
@@ -184,3 +194,59 @@ async fn get(db_conn: DbConnection) -> superposition::Result<Json<Vec<DefaultCon
     let result: Vec<DefaultConfig> = default_configs.get_results(&mut conn)?;
     Ok(Json(result))
 }
+
+pub fn check_key_in_context_overrides(
+    key: &String,
+    conn: &mut PooledConnection<ConnectionManager<PgConnection>>,
+) -> superposition::Result<Vec<String>> {
+    let result: Vec<Context> = contexts.load(conn).map_err(|err| {
+        log::error!("failed to fetch contexts with error: {}", err);
+        db_error!(err)
+    })?;
+
+    let mut context_ids = vec![];
+    for context in result.iter() {
+        from_value::<Map<String, Value>>(context.override_.to_owned())
+            .map_err(|err| {
+                log::error!("failed decode override into object: {}", err);
+                unexpected_error!(err)
+            })?
+            .get(key)
+            .map_or((), |_| context_ids.push(context.id.to_owned()))
+    }
+    Ok(context_ids)
+}
+
+#[delete("/{key}")]
+async fn delete_default_config(
```

Comment:
```suggestion
async fn delete(
```

Reply:
**delete** name collides with the diesel::delete function , thats why kept this as delete_default_config